src/other-scripts/recaptcha/index.js

```diff
@@ -147,8 +152,16 @@ function renderWidget( form, onSuccess = null, onError = null ) {
 			...options,
 			callback: successCallback,
 			'error-callback': () => {
-				refreshWidget( button );
-				const message = wp.i18n.__( 'There was an error with reCAPTCHA. Please try again.', 'newspack-plugin' );
+				const retryCount = parseInt( button.getAttribute( 'data-recaptcha-retry-count' ) ) || 0;
+				if ( retryCount < 3 ) {
+					button.setAttribute( 'data-recaptcha-retry-count', retryCount + 1 );
+					refreshWidget( button );
+				} else {
+					clearInterval( refreshIntervalId );
+				}
+				const message = retryCount < 3
+					? wp.i18n.__( 'There was an error with reCAPTCHA. Please try again.', 'newspack-plugin' )
+					: wp.i18n.__( 'There was an error with reCAPTCHA. Please reload the page and try again.', 'newspack-plugin' );
```

Comment:
I wonder if we should instead nudge the reader to contact the site admin instead. I have seen cases where the API fails for other unknown reasons though (possibly due to some dev site strangeness), so am just asking for a reload.

Let me know if the site admin messaging would be better though!

Reply:
This should be fine. This will happen on any page load so the admins would become aware instantly.